src/app/pages/FrontPage/index.jsx

```diff
@@ -102,30 +101,28 @@ const FrontPageContainer = ({ pageData, mostReadEndpointOverride }) => {
         openGraphType="website"
       />
       <LinkedData type="WebPage" seoTitle={seoTitle} />
-      <StyledFrontPageMain role="main">
```

Comment:
This change removes the `main` from the Front Page, and we need it. Can you please add one?

Reply:
Returned `<StyledFrontPageMain role="main">` to the page